src/applications/disability-benefits/686c-674/config/chapters/veteran-information/veteran-information/veteran-information.js

```diff
@@ -1,65 +1,13 @@
-import merge from 'lodash/merge';
-import ssnUI from 'platform/forms-system/src/js/definitions/ssn';
-import currentOrPastDateUI from 'platform/forms-system/src/js/definitions/currentOrPastDate';
 import { veteranInformation } from '../../../utilities';
+import VeteranInformationComponent from './VeteranInformationComponent.js';
 
 export const schema = {
-  type: 'object',
-  properties: {
-    veteranInformation: {
-      ...veteranInformation.properties.veteranInformation,
-    },
-  },
+  ...veteranInformation.properties.veteranInformation,
 };
 
 export const uiSchema = {
-  veteranInformation: {
-    fullName: {
-      first: {
-        'ui:title': 'Your first name',
-        'ui:required': () => true,
-      },
-      middle: {
-        'ui:title': 'Your middle name',
-      },
-      last: {
-        'ui:title': 'Your last name',
-        'ui:required': () => true,
-      },
-      suffix: {
-        'ui:title': 'Suffix',
-        'ui:options': {
-          widgetClassNames: 'usa-input-medium',
-        },
-      },
-    },
-    ssn: {
-      ...ssnUI,
-      ...{
-        'ui:title': 'Your Social Security number',
-        'ui:required': () => true,
-      },
-    },
-    vaFileNumber: {
-      'ui:title': 'Your VA file number',
-      'ui:errorMessages': {
-        pattern: 'Please enter only numbers',
-      },
-      'ui:options': {
-        widgetClassNames: 'usa-input-medium',
-      },
-    },
-    serviceNumber: {
-      'ui:title': 'Your service number',
-      'ui:errorMessages': {
-        pattern: 'Please enter only numbers',
-      },
-      'ui:options': {
-        widgetClassNames: 'usa-input-medium',
-      },
-    },
-    birthDate: merge(currentOrPastDateUI('Your date of birth'), {
-      'ui:required': () => true,
-    }),
+  'ui:description': VeteranInformationComponent,
+  'ui:options': {
+    hideOnReview: true,
```

Comment:
Do we really want to hide this information on the review page? Can we confirm with Jim?

Reply:
yep, I confirmed with Jim and he wants this hidden on the review page